src/components/brodsmuler/brodsmuler.tsx

```diff
@@ -1,24 +1,33 @@
 import './brodsmuler.less'
 
+import { OppChevron } from 'nav-frontend-chevron'
 import Lenke from 'nav-frontend-lenker'
 import { Normaltekst } from 'nav-frontend-typografi'
-import React, { useState } from 'react'
+import React, { useEffect, useRef, useState } from 'react'
 import { Link } from 'react-router-dom'
 
 import { Brodsmule } from '../../types/types'
 import env from '../../utils/environment'
+import Vis from '../vis'
 import personIkon from './person.svg'
 
-const BrodsmuleBit = ({ sti, tittel, sisteSmule, erKlikkbar }: Brodsmule) => {
-    const erEkstern = sti && sti.includes(process.env.REACT_APP_SYKEFRAVAER_CONTEXT_ROOT!)
+const LITEN = 768
+
+const faste: Brodsmule[] = [
+    { tittel: 'Ditt NAV', sti: '/dittnav', erKlikkbar: true },
+    { tittel: 'Ditt sykefravær', sti: env.sykefravaerUrl, erKlikkbar: true }
+]
+
+const BrodsmuleBit = ({ sti, tittel, erKlikkbar }: Brodsmule) => {
+    const erEkstern = sti && sti.startsWith('http')
```

Comment:
dette funker fortsatt ikke for /dittnav . Du må vel kanskje dra den helt ut og bruke hele urlen som jeg gjorde i spinnsyn-frontend